app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Help" %>
+
+
+<h1>Help</h1>
+<p>
+Get help on the Ruby on Rails Tutorial at the
+<%= link_to "Rails Tutorial help page", "http://www.railstutorial.org/help"%>
+  To get help on this sample app, see the
+<%= link_to "<em>Ruby on Rails Tutorial</em>book", "http://www.railstutorial.org/book"%>
```

Comment:
the static links, you should add to setting.yml file and call them here

Reply:
done